### DISCUSSION TITLE
Baseline

### DISCUSSION THREAD

**Discussion by aaronlsmiles** [decadenza/SimpleStereo#13] [Q&A] [answered]
Changing the `square size` parameter produces only a slightly different RPE, yet the Baseline changes dramatically. Why is this and what exactly is Baseline telling us?

**Comment by decadenza** [comment, marked as answer]
Baseline is the distance between the two cameras, so you change its unit of measure.

The reprojection error is not a dimensional parameter and shouldn't change for different square sizes. Actually it should not change at all (except for maybe machine rounding error).

**Reply by aaronlsmiles** [reply]
When I run the calibration code with the real world square size of my checkerboard (27.0), I get a Baseline: 146.05850579369803

My cameras are exactly 10cm apart. Does this baseline mean there's something not right with my calibration then?

**Reply by decadenza** [reply]
Not necessarily. The baseline is the distance between the two camera centers, which depends on where the rays of the lenses focus. This is not necessarily the centre of the physical camera.

Try to scan an object and check that the dimensions correspond to the real value.

**Reply by aaronlsmiles** [reply]
> Try to scan an object and check that the dimensions correspond to the real value.

How would I go about doing that? Would this be done during 3d reconstruction step?
